index.js

```diff
@@ -18,7 +18,10 @@ const exit = msg => {
 /**
  * Executes the provided shell command and redirects stdout/stderr to the console
  */
-const run = (cmd, cwd) => execSync(cmd, { encoding: "utf8", stdio: "inherit", cwd });
+const run = (cmd, cwd) => {
+  console.log(`+ ${cmd}`);
+  execSync(cmd, { encoding: "utf8", stdio: "inherit", cwd });
+}
```

Comment:
I'd rather not fill the logs with all executed commands. The action already provides user-friendly logs.